usr/share/rear/finalize/Fedora/i386/550_rebuild_initramfs.sh

```diff
@@ -58,10 +58,10 @@ NEW_INITRD_MODULES=( ${OLD_INITRD_MODULES[@]} ${OLD_INITRD_MODULES[@]} $( cat $T
 # uniq INITRD_MODULES
 NEW_INITRD_MODULES=( $(tr " " "\n" <<< "${NEW_INITRD_MODULES[*]}" | sort | uniq -u) )
 
-Log "New INITRD_MODULES='${OLD_INITRD_MODULES[@]} ${NEW_INITRD_MODULES[@]}'"
+Log "New INITRD_MODULES are ${OLD_INITRD_MODULES[*]} ${NEW_INITRD_MODULES[*]}"
 INITRD_MODULES="${OLD_INITRD_MODULES[@]} ${NEW_INITRD_MODULES[@]}"
 
-WITH_INITRD_MODULES=$( printf '%s\n' ${INITRD_MODULES[@]} | awk '{printf "--with=%s ", $1}' )
+WITH_INITRD_MODULES=$( printf '%s\n' ${INITRD_MODULES[*]} | awk '{printf "--with=%s ", $1}' )
```

Comment:
`INITRD_MODULES` is not even an array, because the assignment above does not use `( )`.
Try it:
```
$ OLD_INITRD_MODULES=( a b c )
$ NEW_INITRD_MODULES=( d e f )
$ INITRD_MODULES="${OLD_INITRD_MODULES[@]} ${NEW_INITRD_MODULES[@]}"
$ for i in "${INITRD_MODULES[@]}"; do echo $i; done
a b c d e f
```
note the single line of output, instead of six lines.

Reply:
https://github.com/rear/rear/pull/2771/commits/2fbb8ed8230e4af972d3a5933c6505c852950866
handles INITRD_MODULES and WITH_INITRD_MODULES as strings